src/App.tsx

```diff
@@ -28,6 +28,7 @@ import { RedirectPathToSwapOnly, RedirectToSwap } from './views/Swap/redirects'
 // Route-based code splitting
 // Only pool is included in the main bundle because of it's the most visited page
 const Home = lazy(() => import('./views/Home'))
+const Home2 = lazy(() => import('./views/Home2'))
```

Comment:
I guess this is temporary, but I'll note it just in case.
The old home folder needs to be removed.

Reply:
🗑️